src/esp/gfx/RenderTarget.cpp

```diff
@@ -215,18 +223,23 @@ struct RenderTarget::Impl {
 
   void readFrameObjectId(const Mn::MutableImageView2D& view) {
     CORRADE_ASSERT(
-        flags_ & Flag::ObjectIdBuffer,
+        flags_ & Flag::ObjectIdTexture,
         "RenderTarget::Impl::readFrameObjectId(): this render target "
-        "was not created with objectId render buffer enabled.", );
-
-    framebuffer_.mapForRead(ObjectIdBuffer).read(framebuffer_.viewport(), view);
+        "was not created with objectId render texture enabled.", );
+#ifdef MAGNUM_TARGET_WEBGL
+    framebuffer_.mapForRead(ObjectIdTextureColorAttachment)
+        .read(framebuffer_.viewport(), view);
+#else
+    objectIdTexture_.image(0, view);
```

Comment:
@mosra: I used the `image(...)` here. Is it necessary?

Reply:
Why not have the same code path for both, like it was before? I don't think there should be any practical difference between the two, so in that case I'd say less code is better code :D